### PR TITLE
Move GitHub API URL to .env

### PR DESCRIPTION
**Pull request by KBandipo**
This pull request refactors the code to use an environment variable for the GitHub API base URL. These were the changes made

- Added GITHUB_API_URL to the .env.
- Updated server.js to use the base URL from the environment variable for API requests and merge with the endpoints (/issues and /pulls).
- Updated the .env.example file to reflect the new GITHUB_API_URL variable.

### CODE REVIEW

**Comment by KBandipo** [comment]
Hello @collabo-community/review-web-app, my PR is ready for review. Thank you

File: server.js [hunk 1]
```diff
@@ -11,10 +11,9 @@ app.use(express.static("public")); // Serve static files from the 'public' direc
 // Web-app-fetching-pull-request-from-GitHub
 
 const PORT = process.env.PORT || 3000;
-const GITHUB_ISSUES_API_URL =
-  "https://api.github.com/repos/KBandipo/GitHub-REST-API/issues";
-const GITHUB_PR_API_URL =
-  "https://api.github.com/repos/KBandipo/GitHub-REST-API/pulls";
+const GITHUB_API_URL = process.env.GITHUB_API_URL;
+const GITHUB_ISSUES_API_URL = `${GITHUB_API_URL}/issues`;
+const GITHUB_PR_API_URL = `${GITHUB_API_URL}/pulls`;
```
Comment (Ifycode) [inline server.js:16]: Nice try @KBandipo 
There's no need to store them again in variables in lines 14-16.

You can remove lines 14 - 16 from the code, the apply the .env variable directly like so:

````
const response = await fetch(`${process.env.GITHUB_API_URL}/issues`
````

Same for /pulls too.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (KBandipo) [reply server.js:16]: Hummmh, keep it DRY (Don't Repeat Yourself),  thanks boss. That's why it's good to work with senior dev

**Comment by KBandipo** [comment]
Hello @collabo-community/review-web-app, my PR is ready; I have refactored the code

**Review by Ifycode — APPROVED** [review]

**Pull request merged**